### PR TITLE
[FR] Add osu! Talk Event: Overcoming Obstacles

### PR DESCRIPTION
**Pull request by DoryanR**
Proposal to add the French translation of the wiki page `Community\osu!_Talk_Event\Overcoming_Obstacles`.

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
- [x] The changes have been approved on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+Pour participer en tant que speaker à une prochaine itération de l'événement, ou recommander quelqu'un qui serait apte à donner une conférence, remplissez le [formulaire de nomination d'un speaker](https://forms.gle/HCD6ac8JwURGh8zx8).
+
+## Informations sur l'événement
+
+- **L'événement durera 3 jours au total**, du 20 au 22 mai 2022.
+- **Le speaker utilisera le salon Scène Discord** dans le [serveur Discord osu!dev](https://discord.gg/ppy) pour faire leur exposé.
+- **Les interventions dureront moins de 18 minutes.** Il s'agit de la durée maximale autorisée, et non d'un objectif à atteindre - même un exposé de trois minutes peut avoir un impact considérable.
+- Les histoires partagées ne doivent pas nécessairement concerner l'université d'Ottawa, mais comme il s'agit d'un événement lié à l'université, il est recommandé aux speakers d'essayer de relier leur histoire ou la leçon qu'elle contient à l'université d'Ottawa d'une manière ou d'une autre.
+
+## Links
```
Comment (DoryanR) [inline wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md:25]:
```suggestion
## Liens
```

File: wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+# osu! Talk Event: Overcoming Obstacles
+
+![Bannière de osu! Talk Event: Overcoming Obstacles](/wiki/shared/news/2022-05-19-osu-talk-event-overcoming-obstacles/ote-newspost-banner.png)
+
+L'**osu! Talk Event** est un événement organisé par osu! University où différentes personnes ont tenu des conférences sur des sujets variés. Il s'agissait de la première itération de l'événement dont le thème était "surmonter les obstacles".
+
+Les intervenants ont tenu leurs discours dans un salon Stage sur le [serveur Discord osu!dev](/wiki/Community/osu!dev_Discord_server). Pendant l'événement, des salons de discussion ont été ouverts dans le salon `#community-meetings` pour les discussions.
+
+Les conférences ont également été diffusées en direct sur [la chaîne Twitch de l'Université d'Ossétie du Sud](https://twitch.tv/osuuniversity) et des enregistrements de chaque conférence sont disponibles sur [la chaîne YouTube de l'Université d'Ossétie du Sud](https://www.youtube.com/c/osuuniversity).
```
Comment (Walavouchey) [inline wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md:14]: > l'Université d'Ossétie du Sud

?

Comment (DoryanR) [inline wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md:14]:
```suggestion
Les conférences ont également été diffusées en direct sur [la chaîne Twitch de osu! University](https://twitch.tv/osuuniversity) et des enregistrements de chaque conférence sont disponibles sur [la chaîne YouTube de osu! University](https://www.youtube.com/c/osuuniversity).
```

File: wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+Pour participer en tant que speaker à une prochaine itération de l'événement, ou recommander quelqu'un qui serait apte à donner une conférence, remplissez le [formulaire de nomination d'un speaker](https://forms.gle/HCD6ac8JwURGh8zx8).
+
+## Informations sur l'événement
+
+- **L'événement durera 3 jours au total**, du 20 au 22 mai 2022.
+- **Le speaker utilisera le salon Scène Discord** dans le [serveur Discord osu!dev](https://discord.gg/ppy) pour faire leur exposé.
+- **Les interventions dureront moins de 18 minutes.** Il s'agit de la durée maximale autorisée, et non d'un objectif à atteindre - même un exposé de trois minutes peut avoir un impact considérable.
+- Les histoires partagées ne doivent pas nécessairement concerner l'université d'Ottawa, mais comme il s'agit d'un événement lié à l'université, il est recommandé aux speakers d'essayer de relier leur histoire ou la leçon qu'elle contient à l'université d'Ottawa d'une manière ou d'une autre.
```
Comment (Walavouchey) [inline wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md:23]: here too

Comment (DoryanR) [inline wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md:23]:
```suggestion
- Les histoires partagées ne doivent pas nécessairement concerner osu! University, mais comme il s'agit d'un événement lié à l'université, il est recommandé aux speakers d'essayer de relier leur histoire ou la leçon qu'elle contient à l'université d'Ottawa d'une manière ou d'une autre.
```

File: wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+Pour participer en tant que speaker à une prochaine itération de l'événement, ou recommander quelqu'un qui serait apte à donner une conférence, remplissez le [formulaire de nomination d'un speaker](https://forms.gle/HCD6ac8JwURGh8zx8).
+
+## Informations sur l'événement
+
+- **L'événement durera 3 jours au total**, du 20 au 22 mai 2022.
+- **Le speaker utilisera le salon Scène Discord** dans le [serveur Discord osu!dev](https://discord.gg/ppy) pour faire leur exposé.
+- **Les interventions dureront moins de 18 minutes.** Il s'agit de la durée maximale autorisée, et non d'un objectif à atteindre - même un exposé de trois minutes peut avoir un impact considérable.
+- Les histoires partagées ne doivent pas nécessairement concerner osu! University, mais comme il s'agit d'un événement lié à l'université, il est recommandé aux speakers d'essayer de relier leur histoire ou la leçon qu'elle contient à l'université d'Ottawa d'une manière ou d'une autre.
```
Comment (Walavouchey) [inline wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md:23]:
```suggestion
- Les histoires partagées ne doivent pas nécessairement concerner osu!, mais comme il s'agit d'un événement lié à osu!, il est recommandé aux speakers d'essayer de relier leur histoire ou la leçon qu'elle contient à osu! d'une manière ou d'une autre.
```

Comment (Walavouchey) [reply wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md:23]: where did these even come from... ?

File: wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+  - osu! University
+---
+
+# osu! Talk Event: Overcoming Obstacles
+
+![Bannière de osu! Talk Event: Overcoming Obstacles](/wiki/shared/news/2022-05-19-osu-talk-event-overcoming-obstacles/ote-newspost-banner.png)
+
+L'**osu! Talk Event** est un événement organisé par osu! University où différentes personnes ont tenu des conférences sur des sujets variés. Il s'agissait de la première itération de l'événement dont le thème était "surmonter les obstacles".
+
+Les intervenants ont tenu leurs discours dans un salon Stage sur le [serveur Discord osu!dev](/wiki/Community/osu!dev_Discord_server). Pendant l'événement, des salons de discussion ont été ouverts dans le salon `#community-meetings` pour les discussions.
```
Comment (Ganondorf-1) [inline wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md:12]:
```suggestion
Les intervenants ont tenu leurs discours dans un salon Étape sur le [serveur Discord osu!dev](/wiki/Community/osu!dev_Discord_server). Pendant l'événement, des fils de discussion ont été ouverts dans le salon `#community-meetings` pour en discuter.
```

File: wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+Les conférences ont également été diffusées en direct sur [la chaîne Twitch de osu! University](https://twitch.tv/osuuniversity) et des enregistrements de chaque conférence sont disponibles sur [la chaîne YouTube de osu! University](https://www.youtube.com/c/osuuniversity).
+
+Pour participer en tant que speaker à une prochaine itération de l'événement, ou recommander quelqu'un qui serait apte à donner une conférence, remplissez le [formulaire de nomination d'un speaker](https://forms.gle/HCD6ac8JwURGh8zx8).
```
Comment (Ganondorf-1) [inline wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md:16]:
```suggestion
Pour participer en tant qu'intervenant à une prochaine itération de l'événement, ou recommander quelqu'un qui serait apte à donner une conférence, remplissez le [formulaire de nomination des intervenants](https://forms.gle/HCD6ac8JwURGh8zx8).
```

File: wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+## Informations sur l'événement
+
+- **L'événement durera 3 jours au total**, du 20 au 22 mai 2022.
+- **Le speaker utilisera le salon Scène Discord** dans le [serveur Discord osu!dev](https://discord.gg/ppy) pour faire leur exposé.
```
Comment (Ganondorf-1) [inline wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md:21]:
```suggestion
- **L'intervenant utilisera le salon Étape de Discord** dans le [serveur Discord osu!dev](https://discord.gg/ppy) pour faire leur conférence.
```

File: wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+- **Les interventions dureront moins de 18 minutes.** Il s'agit de la durée maximale autorisée, et non d'un objectif à atteindre - même un exposé de trois minutes peut avoir un impact considérable.
```
Comment (Ganondorf-1) [inline wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md:22]:
```suggestion
- **Les conférences dureront moins de 18 minutes.** Il s'agit de la durée maximale autorisée, et non d'un objectif à atteindre - même une conférence de trois minutes peut avoir un impact considérable.
```

File: wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+- Les histoires partagées ne doivent pas nécessairement concerner osu!, mais comme il s'agit d'un événement lié à osu!, il est recommandé aux speakers d'essayer de relier leur histoire ou la leçon qu'elle contient à osu! d'une manière ou d'une autre.
```
Comment (Ganondorf-1) [inline wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md:23]:
```suggestion
- Les histoires partagées ne doivent pas nécessairement concerner osu!, mais comme il s'agit d'un événement lié à osu!, il est recommandé aux intervenants d'essayer de relier leur histoire ou la leçon qu'elle contient à osu! d'une manière ou d'une autre.
```

File: wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+
+## Liens
+
+- [Fil de discussion](https://osu.ppy.sh/community/forums/topics/1574152)
+- [News post de l'annonce](https://osu.ppy.sh/home/news/2022-05-19-osu-talk-event-overcoming-obstacles)
+- [Serveur Discord osu!dev](/wiki/Community/osu!dev_Discord_server)
+- [Serveur Discord de osu! University](https://discord.gg/QubdHdnBVg)
+- [Chaîne Twitch de osu! University](https://twitch.tv/osuuniversity)
+- [Chaîne YouTube de osu! University](https://www.youtube.com/c/osuuniversity)
```
Comment (Ganondorf-1) [inline wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md:32]:
```suggestion
- [Chaîne YouTube d'osu! University](https://www.youtube.com/c/osuuniversity)
```

File: wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+- [Formulaire de nomination des speakers](https://forms.gle/HCD6ac8JwURGh8zx8)
+
```
Comment (Ganondorf-1) [inline wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md:34]:
```suggestion
- [Formulaire de nomination des intervenants](https://forms.gle/HCD6ac8JwURGh8zx8)
```

File: wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+## Programme
+
+### Vendredi 20 mai 2022
+
+**[Pour voir la vidéo complète de cette journée de l'événement, cliquez ici.](https://www.youtube.com/watch?v=dXwMiPBiQpk)**
+
+| Heure (UTC) | Speaker | Titre | Vidéo |
```
Comment (Ganondorf-1) [inline wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md:41]:
```suggestion
| Heure (UTC) | Intervenant | Titre | Vidéo |
```

File: wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md [hunk 13]
```diff
@@ -0,0 +1,69 @@
...
+| 21:30 | [PandaPasBo](https://osu.ppy.sh/users/10262231) | "Why expectation ruins your skill in osu" | [#1](https://www.youtube.com/watch?v=ES-gjqgu0i8) |
+| 22:00 | [xootynator](https://osu.ppy.sh/users/3717598) | "xooty's left hand training" | [#1](https://www.youtube.com/watch?v=vkEw7YF_8fA) |
+| 22:30 | [Likean00b](https://osu.ppy.sh/users/4860447) | "When tournaments become a hindrance" | [#1](https://www.youtube.com/watch?v=Ll5itEiaZko) |
+| 23:00 | [MakiDonalds](https://osu.ppy.sh/users/11610772) | "Break your walls before they break you" | [#1](https://www.youtube.com/watch?v=NU6JRqGrEGg) |
+
+### Samedi 21 mai 2022
+
+**[Pour voir la vidéo complète de cette journée de l'événement, cliquez ici.](https://www.youtube.com/watch?v=YLaFZup2mFo)**
+
+| Heure (UTC) | Speaker | Titre | Vidéo |
```
Comment (Ganondorf-1) [inline wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md:52]:
```suggestion
| Heure (UTC) | Intervenant | Titre | Vidéo |
```

File: wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md [hunk 14]
```diff
@@ -0,0 +1,69 @@
...
+| 13:00 | [Axiqn](https://osu.ppy.sh/users/21130016) | "Focusing on weaknesses" | [#1](https://www.youtube.com/watch?v=pGBBxAM7QkU) |
+| 13:30 | [UserAlsoExists](https://osu.ppy.sh/users/19036931) | "The "play less" meta" | [#1](https://www.youtube.com/watch?v=qQvVtu19CnM) |
+| 14:30 | [-Liability](https://osu.ppy.sh/users/12260184) | "Doubting yourself" | [#1](https://www.youtube.com/watch?v=_dGxynP9HSs) |
+| 15:00 | [Astral52](https://osu.ppy.sh/users/11936432) | "Opening the mind to have fun: How to build a less toxic mentality" | [#1](https://www.youtube.com/watch?v=hJUwU8YLUUg) |
+
+### Dimanche 22 mai 2022
+
+**[Pour voir la vidéo complète de cette journée de l'événement, cliquez ici.](https://www.youtube.com/watch?v=TICCyoj1F1U)**
+
+| Heure (UTC) | Speaker | Titre | Vidéo |
```
Comment (Ganondorf-1) [inline wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md:64]:
```suggestion
| Heure (UTC) | Intervenant | Titre | Vidéo |
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

File: wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md [hunk 15]
```diff
@@ -0,0 +1,68 @@
...
+## Liens
+
+- [Fil de discussion](https://osu.ppy.sh/community/forums/topics/1574152)
+- [News post de l'annonce](https://osu.ppy.sh/home/news/2022-05-19-osu-talk-event-overcoming-obstacles)
+- [Serveur Discord osu!dev](/wiki/Community/osu!dev_Discord_server)
+- [Serveur Discord osu! University](https://discord.gg/QubdHdnBVg)
+- [Chaîne Twitch d'osu! University](https://twitch.tv/osuuniversity)
+- [Chaîne YouTube d'osu! University](https://www.youtube.com/c/osuuniversity)
+- [Formulaire de nomination des intervenants](https://forms.gle/HCD6ac8JwURGh8zx8)
+## Programme
```
Comment (DoryanR) [inline wiki/Community/osu!_Talk_Event/Overcoming_Obstacles/fr.md:34]:
```suggestion
- [Formulaire de nomination des intervenants](https://forms.gle/HCD6ac8JwURGh8zx8)

## Programme
```

**Review by Biokax — APPROVED** [review]

**Review by cl8n — APPROVED** [review]

**Pull request merged**